### DISCUSSION TITLE
Inline documentation (PHPDoc)

### DISCUSSION THREAD

**Discussion by OvalMedia** [filamentphp/filament#4726] [Ideas]
Regarding future upgrades I would suggest to add docblocks (https://en.wikipedia.org/wiki/PHPDoc) to the code.
I am using PHPStorm and coming from an ecommerce environment where this is a standard it was quite confusing browsing your code. It looked "naked". And usually a good inline documentation often gets you to the point quicker. Also I am sure it would help reduce the amount of support requests.

I understand if you are not used to this overhead this seems tedious. I used to think that way but nowadays I would not want to miss it (which I do here).

**Comment by jhm-ciberman** [comment]
+1 for this motion. PHP Docs establishes a contract and define what a method does. Also they are helpful for understanding the API and exploring the features.
